### PR TITLE
Display monitored resources in the kinto-changes capability.

### PR DESCRIPTION
**Pull request by Natim**
r? @leplatrem

### CODE REVIEW

File: kinto_changes/__init__.py [hunk 1]
```diff
@@ -1,8 +1,15 @@
+from pyramid.settings import aslist
+
+
 def includeme(config):
+    settings = config.get_settings()
+    resources = aslist(settings.get('event_listeners.changes.collections', []))
```
Comment (Natim) [inline kinto_changes/__init__.py]: Because people can configure it with another name the `event_listeners.changes` do we have another way of doing this?

Comment (leplatrem) [inline kinto_changes/__init__.py]: Quick alternatives I see:

* Look for `kinto_changes` in `use` settings
* Do not use a listener from settings but configure a subscriber instead

File: kinto_changes/__init__.py [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
+
     config.add_api_capability(
         "changes",
         description="Track modifications of records in Kinto and store"
                     " the collection timestamps into a specific bucket"
                     " and collection.",
         url="http://kinto.readthedocs.io/en/latest/api/1.x/"
-            "synchronisation.html#polling-for-remote-changes")
+            "synchronisation.html#polling-for-remote-changes",
+        resources=resources)
```
Comment (leplatrem) [inline kinto_changes/__init__.py]: Why do talk about resources if the setting is called `collections` ?

Comment (Natim) [inline kinto_changes/__init__.py]: I did it like that because it is called like that in kinto-signer but I can change it :)

**Pull request merged**